packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md

```diff
@@ -0,0 +1,377 @@
+<dt><strong>Features</strong></dt>
+<dd>Distinct pieces of functionality. Features are bundled within products but may also be offered separately, for example, Early Access features.</dd>
+</dl>
+
+### Aerial sandbox and production environments
+
+Okta provides the following environments to use with your Aerial account:
+
+- **Sandbox**: Only connected to OP3 cells. Use for scale testing and repetitive testing.
+
+  `https://aerial-sandbox.okta.com`
+
+- **Production**:  Connected to `okta.com` and `okta.preview.com`. Use to explore the Aerial API, or test net-new development. Link a preview org to your production instance.
+
+  `https://aerial-apac.okta.com`
+
+See [Link the new org to the Aerial account](#link-the-org-to-the-aerial-account).
+
+### Summary of steps
+
+Follow these steps to create and configure new orgs in your Aerial account:
+
+1. [Authenticate with Okta Aerial](#authenticate-with-okta-aerial): Configure an API client in the Aerial org to call the Okta Aerial API.
+1. [Create and configure a child org](#create-a-child-org): Create a child org based on the parent org using the Org creator API.
+1. [Link the org to the Aerial account](#link-the-org-to-the-aerial-account): Link the org to the Aerial account using the Aerial API.
+1. [Enable products in the org](#enable-products-in-the-org): Enable products in the Org using the Aerial API.
+1. [Configure the org](#configure-the-org): Configure the org using Okta objects in the Aerial API.
+
+You can also manage an org's status, and enable or disable products in an org:
+
+- [Manage an org's status](#manage-an-orgs-status)
+- [List the configured products in the Org](#list-the-configured-products)
+- [Enable a product in an Org](#enable-a-product-in-an-org)
+- [Remove a product from an Org](#remove-a-product-from-an-org)
+
+<div class="full">
+
+![Okta Aerial - summary of steps](/img/aerial-full-flow.png)
+
+</div>
+
+## Register an Okta Aerial API client
+
+Use an Aerial API client to add orgs to the Aerial account and to modify products.
+
+Only a super admin in your Aerial org can grant scopes to the client. Configure scopes once per API client.
+
+In your Aerial org, create an API client:
+
+1. In the **Admin Console**, go to **Applications** > **Applications**, and then click **Create App Integration**.
+1. Select **API Services** as the sign-in method, and click **Next**.
+1. Enter a name for your client and click **Save**.
+
+Configure the signing keys for the client:
+
+1. In the **Client Credentials** section of the **General** tab, click **Edit** to change the client authentication method.
+1. Select **Public key/Private key** as the client authentication method.
+1. Choose either **Save keys in Okta** or **Use a URL to fetch keys dynamically**:
+   - If you want to save keys in Okta, click **Add key**.
+   - If you want to use a URL to fetch keys dynamically, you need to provide a URL that returns the [JWKS documents](https://www.rfc-editor.org/rfc/rfc7517#section-5).
```

Comment:
You might want to link to my document Client secret rotation and key management - it covers the steps for saving the keys in Okta as well as using a URL to fetch keys dynamically:

https://developer.okta.com/docs/guides/client-secret-rotation-key/main/